_locales/en/messages.json

```diff
@@ -1949,7 +1949,7 @@
 		"message": "No, Keep History"
 	},
 	"panel_stats_pitch_modal_text": {
-		"message": "Unlock your historical stats (and other cool perks) by upgrading to Ghostery Plus for $SPAN_START$&#36;2/month.$SPAN_END$$LINK_START$Learn More$LINK_END$",
+		"message": "Unlock your historical stats (and other cool perks) by upgrading to Ghostery Plus$LINK_START$Learn More$LINK_END$",
```

Comment:
Can you add the period back in?

Reply:
Added back in--I still can't believe you caught that